docs/source/tagset/elemento-element-citation.rst

```diff
@@ -64,16 +61,15 @@ Os valores possíveis para o atributo ``@publication-type`` são:
 
 .. note::
 
-  * Nunca incluir informação com formatação ``inline`` (``<italic>``, ``<bold>`` etc.)
-    dentro de um elemento;
+  * Nunca incluir informação com formatação ``inline`` (``<italic>``, ``<bold>`` etc.) dentro de um elemento;
   * Evitar pontuação (ponto final, vírgula etc) dentro de :ref:`elemento-element-citation`;
```

Comment:
Infelizmente, @leticiaquino, há ocorrência de pontuação no elemento ``<article-title>`` dentro de ``<element-citation>``. No próprio exemplo dado, ocorrem ``':'`` e ``'?'``, que são pontuações válidas para títulos de artigos. Outros podem ocorrer (``';','-','!'``). Creio que o nunca seria contraditório, não? No segundo exemplo, <publisher-name> contém uma entidade (``&amp;``) e entidades contém pontuação. Isso pode dar um nó na cabeça do usuário. Exceto se for criado um parágrafo dizendo das ocorrências permitidas de caracteres dentro de alguns elementos (títulos de artigo, de seção, publicador etc).